third-culture-software/bhima
# Added Euros and fixed exchange rate selection page

Added support for Euros and added related currency and exchange rate tests.

**WARNINGS**
- This PR includes migration code.  When migrating production sites, you should probably update the default exchange rate for Euros to be a more current value.

**TESTING**
- Use bhima_test
- Go to page:  Administration / Exchange Rate
- Notice the new entry for Euros
- Click on the [Update Exchange Rate(s)] button to get the modal for changing the exchange rate
- Select Euros from the drop-down currency selector
- Notice that the current exchange rate is updated for Euros (just below the date field)
- Try changing the value and submitting
- Notice the new value is shown on the main page
- Go back into the modal, select Euros again and verify that they new value is shown

**NOTES**
- The process of adding a new currency requires multiple steps and should be documented in the Wiki.
